scripts/zones/Attohwa_Chasm/globals.lua

```diff
@@ -0,0 +1,39 @@
+-----------------------------------
+-- Zone: Attohwa_Chasm
+-- Desc: this file contains functions that are shared by multiple luas in this zone's directory
+-----------------------------------
+local ID = zones[xi.zone.ATTOHWA_CHASM]
+-----------------------------------
+
+local attohwaChasmGlobal =
+{
+    --[[..............................................................................................
```

Comment:
Lets not use this style of comments, these are horrifying 🤢

Reply:
> Lets not use this style of comments, these are horrifying 🤢

Yep, I was unsure if that was used in many globals for some reason :) , have changed the style, thanks!